### PR TITLE
Homework #7 is completed.

### PR DESCRIPTION
**Pull request by komyagin**

### CODE REVIEW

File: src/main/java/hw7/entities/MetalAndColors.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    private static final String LINE_SEPARATOR = ", ";
+
+    private int oddSummaryNumber;
+    private int evenSummaryNumber;
+    private List<String> elements;
+    private String color;
+    private String metal;
+    private List<String> vegetables;
+
+    public MetalAndColors() {
```
Comment (troe88) [inline src/main/java/hw7/entities/MetalAndColors.java]: What is the reason of NoArgsConstructor here ??

File: src/main/java/hw7/pages/BasePage.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    private LoginForm loginForm;
+
+    @Css("#user-name")
+    private Label userName;
+
+    @Css("#user-icon")
+    private Button userIcon;
+
+    @XPath("//ul[@class='uui-navigation nav navbar-nav m-l8']/li")
```
Comment (troe88) [inline src/main/java/hw7/pages/BasePage.java]: This locator can be simplified.

**Review by troe88 — APPROVED** [review]

File: src/main/java/hw7/pages/BasePage.java (hunk 2 above)
Comment (komyagin) [reply src/main/java/hw7/pages/BasePage.java]: Simplified to @Css(".m-l8 > li")

File: src/main/java/hw7/entities/MetalAndColors.java (hunk 1 above)
Comment (komyagin) [reply src/main/java/hw7/entities/MetalAndColors.java]: There is such error without NoArgsConstructor:
Error:(9, 49) java: constructor MetalAndColors in class hw7.entities.MetalAndColors cannot be applied to given types;
  required: int,int,java.util.List<java.lang.String>,java.lang.String,java.lang.String,java.util.List<java.lang.String>
  found: no arguments
  reason: actual and formal argument lists differ in length

And I can't use  set() from JdiDataClass which my Class MetalAndColors extends without NoArgConstrutor. 

Added Lombok annotation NoArgsConstructor

**Pull request merged**